CHANGELOG.md

```diff
@@ -1,5 +1,81 @@
+* bug preventing decimals in (-1, 1) from being inserted / queried ([#8720](https://github.com/confluentinc/ksql/pull/8720)) ([de8284a](https://github.com/confluentinc/ksql/commit/de8284a127de323567276eb6daa2f963a30a6352))
+* check for the queryId in the Task ID for registering saturation metric ([#8789](https://github.com/confluentinc/ksql/pull/8789)) ([b53c1de](https://github.com/confluentinc/ksql/commit/b53c1dedcbe44671578362d48e14033129e8b305))
+* clean up mistaken topic names ([#8857](https://github.com/confluentinc/ksql/pull/8857)) ([53ddafe](https://github.com/confluentinc/ksql/commit/53ddafe72dd8b2d284e9575cff856680b4280bba))
+* CLI should return non-zero error code on failure ([#8892](https://github.com/confluentinc/ksql/pull/8892)) ([238f4fd](https://github.com/confluentinc/ksql/commit/238f4fdd929541ccdd4aeee45cd331964f73b9b3))
+* coerce property values to correct type ([#8765](https://github.com/confluentinc/ksql/pull/8765)) ([7f7a076](https://github.com/confluentinc/ksql/commit/7f7a07619a6ed28f022980ad078607502f4a1f8d))
+* decrease default number of shared runtimes ([#8889](https://github.com/confluentinc/ksql/pull/8889)) ([87bc4b2](https://github.com/confluentinc/ksql/commit/87bc4b2b9e96b923a19497e0ccb8dc5b4e5adb91))
+* do not include schema id in session config ([#8869](https://github.com/confluentinc/ksql/pull/8869)) ([5ddb852](https://github.com/confluentinc/ksql/commit/5ddb852afc37939baa72d9cf6e55dfde9d1e6400))
+* empty to add future list ([#8902](https://github.com/confluentinc/ksql/pull/8902)) ([bca79ac](https://github.com/confluentinc/ksql/commit/bca79ac5922c25e2ddb8f0fd78263260babf0a1a))
+* Ensures response end handler is invoked just once ([#8849](https://github.com/confluentinc/ksql/pull/8849)) ([a2efcc5](https://github.com/confluentinc/ksql/commit/a2efcc53f05de45daf0724ff7c4e1467f2548224))
+* Fix bugs in sample standard deviation UDAF ([#8728](https://github.com/confluentinc/ksql/pull/8728)) ([b2f993b](https://github.com/confluentinc/ksql/commit/b2f993b860a4c8075d23f8a77a828eae12af499f))
+* Gives a query completed message for stream pull queries ([#8612](https://github.com/confluentinc/ksql/pull/8612)) ([a44a16b](https://github.com/confluentinc/ksql/commit/a44a16be46d3c61e7f9d8724c4900752e9fc5337))
+* **ksql:** add ifExists/ifNotExist parameters to java client connector functions ([#8851](https://github.com/confluentinc/ksql/pull/8851)) ([eaf2b1f](https://github.com/confluentinc/ksql/commit/eaf2b1f047a126de642f06a0cde30e6c5123762a))
+* **ksql:** allow migrations tool to run connector commands with IF [NOT] EXISTS clauses ([#8855](https://github.com/confluentinc/ksql/pull/8855)) ([a7c8689](https://github.com/confluentinc/ksql/commit/a7c8689a498bba812a9fc6dbc66c8a3b9d814176))
+* logging improvement ([#8715](https://github.com/confluentinc/ksql/pull/8715)) ([2a53f9f](https://github.com/confluentinc/ksql/commit/2a53f9f35474e97600f82ca9f62c874df5321213))

```

Comment:
@wcarlson5 I notice you have a number of "fixes" in this list where the user-impact/relevance is unclear. Do you think these should be included in the changelog or left out? You can leave future changes out of changelogs automatically by using "chore" instead of "fix" for really minor things that aren't worth a changelog note.